vision/product-search/cloud-client/src/main/java/com/example/vision/ProductInProductSetManagement.java

```diff
@@ -135,6 +144,50 @@ public static void removeProductFromProductSet(
   }
   // [END vision_product_search_remove_product_from_product_set]
 
+  // [START vision_product_search_purge_products_in_product_set]
+  /**
+   * Delete all products in a product set.
+   *
+   * @param projectId - Id of the project.
+   * @param location - Region name.
+   * @param productSetId - Id of the product set.
+   * @param force - Perform the purge only when force is set to True.
+   * @throws Exception - any error.
+   */
+  public static void purgeProductsInProductSet(
+          String projectId, String location, String productSetId, boolean force)
```

Comment:
Does `force` need to be passed in or can we just set it to `true` in the Request object with a note there?

Reply:
hmm, thats a good idea, I just converted from Yu-han python version.